src/Sentry/Sentry.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>net5.0;netstandard2.0;net461</TargetFrameworks>
```

Comment:
Probably don't need to target here too. Just let it resolve to ns2.0.
Unless we're changing stuff like using System.Json (while keeping the old targets as they are which would probably surprise some if they retarget their app and without changing this dependency get some different dependencies).